client/components/explorer/explorer-header-directive.html

```diff
@@ -1,6 +1,9 @@
 <div class="pk-header">
 
   <div class="pk-header-toolbar">
+    <span class="pk-header-status" ng-show="ctrl.workQueueSvc.canShowStatus()">
+      {{ ctrl.workQueueSvc.getStatus() }}
```

Comment:
Angular-templated sections can get evaluated a lot; consider storing status as a property and updating the relevant strings on the appropriate events/methods, rather than calculating in the method.